### PR TITLE
Unique keys for v-for

### LINKED ISSUE #11: Other key than index?
These labels won't update after rendering, but maybe index shouldn't be used? Disable 
vue/require-v-for-key? 

https://github.com/coopjakob/kenora-thistles/blob/0a64785d26c3191c8e062d73bbf5276e458177b5/src/components/ProductCard.vue#L49

### PR DESCRIPTION
**Pull request by benjick**
Fixes #11

### CODE REVIEW

File: src/components/ProductCard.vue [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
     <div
       v-for="(theInfo, index) in consumerInfo"
-      :key="index"
+      :key="index + theInfo"
```
Comment (coopjakob) [inline src/components/ProductCard.vue]: `key="theInfo"` should be enough because info is not repeating itself

**Review by coopjakob — COMMENTED** [review]
So, I guess the ID doesn't need to be a short number or string. It can be the full text. Is there a limit?

File: src/components/ProductCard.vue [hunk 2]
```diff
@@ -46,7 +46,7 @@
 
     <div
       v-for="(theInfo, index) in consumerInfo"
```
Comment (coopjakob) [inline src/components/ProductCard.vue]: `error: 'index' is defined but never used`
So it needs to be:
`v-for="theInfo in consumerInfo"`
Is there a proper way to suggest or edit that change and then approve the pull request?

Comment (benjick) [inline src/components/ProductCard.vue]:
```suggestion
      v-for="theInfo in consumerInfo"
```
@coopjakob like this ^ :fire:

Comment (benjick) [reply src/components/ProductCard.vue]:
```
    ```suggestion
          v-for="theInfo in consumerInfo"
    ```
```

**Review by coopjakob — APPROVED** [review]

**Pull request merged**